includes/view/AngelTypes_view.php

```diff
@@ -202,13 +202,21 @@ function AngelType_view_buttons(
     $user_license,
     $user
 ) {
-    if (config('driving_license_enabled') && $angeltype->requires_driver_license) {
+    if (
+        config('driving_license_enabled')
+        && $angeltype->requires_driver_license
+        && ($user_angeltype || auth()->can('user.drive.edit'))
```

Comment:
```suggestion
        && $user_angeltype
```
would be imho enough :D